lib/codemod/src/lib/utils.js

```diff
@@ -1,14 +1,8 @@
 import camelCase from 'lodash/camelCase';
-
-const RESERVED = /^(?:do|if|in|for|let|new|try|var|case|else|enum|eval|false|null|this|true|void|with|await|break|catch|class|const|super|throw|while|yield|delete|export|import|public|return|static|switch|typeof|default|extends|finally|package|private|continue|debugger|function|arguments|interface|protected|implements|instanceof)$/;
-
-export const isReserved = name => RESERVED.exec(name);
+import upperFirst from 'lodash/upperFirst';
 
 export const sanitizeName = name => {
-  let key = camelCase(name);
-  if (isReserved(key)) {
-    key = `${key}Story`;
-  }
+  let key = upperFirst(camelCase(name));
```

Comment:
I wanted to use pipeline operator here, but I'm not sure whether it's OK with other contributors:

```js
let key = name |> camelCase |> upperFirst
```

Reply:
🤔 stage 1 ....

Reply:
Let's not use language features that typescript doesn't support?